src/main/java/com/cincialert/service/UserService.java

```diff
@@ -3,35 +3,30 @@
 import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
I renamed a lot of variables in here to match common naming conventions for JPA.